docker/config/gunicorn_config.py

```diff
@@ -12,19 +12,19 @@
 
 LOGGING_LEVEL = CONFIG("LOGGING_LEVEL", "INFO")
 LOCAL_DEV_ENV = CONFIG("LOCAL_DEV_ENV", False, cast=bool)
-HOST_ID = socket.gethostname()
+HOSTNAME = CONFIG("HOSTNAME", default=socket.gethostname())
 
 
-class AddHostID(logging.Filter):
+class AddHostname(logging.Filter):
     def filter(self, record):
-        record.host_id = HOST_ID
+        record.hostname = HOSTNAME
         return True
```

Comment:
Once we've switched to GCP, we can remove `AddHostname` because we won't need it anymore. I'll write up a follow-up bug for that.

Reply:
why won't need it anymore?

Reply:
because logs in gcp are already tagged with pod and container name?